app/com/arpnetworking/metrics/portal/reports/impl/DatabaseReportRepository.java

```diff
@@ -342,10 +342,9 @@ private ReportSchedule internalModelToBean(final Schedule internalSchedule) {
             beanOneOff.setRunUntil(internalOneOff.getRunUntil().orElse(null));
             return beanOneOff;
         } else if (internalSchedule instanceof NeverSchedule) {
-            final NeverSchedule internalNever = (NeverSchedule) internalSchedule;
             final ReportSchedule beanNever = new NeverReportSchedule();
             beanNever.setRunAt(Instant.ofEpochSecond(0));
-            beanNever.setRunUntil(internalNever.getRunUntil().orElse(null));
+            beanNever.setRunUntil(null);
```

Comment:
I made it so `NeverSchedule` is no longer a subclass of `BoundedSchedule` so it doesn't have a `getRunUntil` method anymore. This should have always been null anyway.